openedx/xblock-lti-consumer
# fix: Fix quote rendering on LTI 1.1 form

This PR:
* Fixes quotes being rendered on LTI 1.1 launch templates by switching to Django template (from Mako template) - they handle escaping characters better.
* Internationalize strings on LTI 1.1 launch page.

Fixes #177.

_This also helps us get rid of the Mako templates._ :grin:

**Testing instructions:**
1. **Use the master branch.**
2. Set up a LTI passport on a course's advanced settings (I'm using the saLTIre test tool - https://saltire.lti.app/tool, no setup required there):
```
[
    "test:jisc.ac.uk:secret"
]
```
3. Add `lti_consumer` to the advanced module list.
4. Create a new section, subsection and unit and add a LTI Consumer block.
5. Set these options:
```
Display name: LTI Consumer "        <- Don't forget the double quote at the end
LTI ID: test
LTI URL: https://saltire.lti.app/tool
```
6. Save and check the output.
![image](https://user-images.githubusercontent.com/27893385/152047261-bd5970b4-5e8f-483e-a186-a338241a301e.png)
_Optionally inspect the parameters: you'll see the missing quote in the name._
7. **Switch to this branch and refresh the page.**
8. Profit.
![image](https://user-images.githubusercontent.com/27893385/152047441-b83d843e-d497-492d-9505-7aaca936493e.png)

**Reviewers:**
- [x] @schenedx
